### DISCUSSION TITLE
Unsupported Linux Version?

### DISCUSSION THREAD

**Discussion by MMStandley** [Security-Onion-Solutions/securityonion#10637] [2.4]
Installed Rocky Linux 9.2 Minimal as noted here: https://docs.securityonion.net/en/2.4/download.html and installed SO 2.4 just fine for a Manager. 

But on a FWD server when I try to run so-setup for 2.4 it errors with this error:

We do not support the version of CentOS you are trying to use.

Any advice?

**Comment by dougburks** [comment]
Here's the 2.4 code that checks Linux version:
https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/setup/so-functions#L944-L974

As you can see, there is no error message that matches `We do not support the version of CentOS you are trying to use.`

Looks like that error message is in the 2.3 version:
https://github.com/Security-Onion-Solutions/securityonion/blob/master/setup/so-functions#L1154

Please try again and make sure you follow the 2.4 instructions at https://docs.securityonion.net/en/2.4/installation.html#installation-on-rocky-linux-or-ubuntu and don't forget the `-b 2.4/main` in the `git clone` command:
```
git clone -b 2.4/main https://github.com/Security-Onion-Solutions/securityonion
```
